### PR TITLE
Schema: Add DcoreEnum value for ArmChina STAR-MC1 processor

### PR DESCRIPTION
**Pull request by JonatanAntoni**

### CODE REVIEW

File: doxygen/src/devices_schema.txt [hunk 1]
```diff
@@ -2370,6 +2370,10 @@ These values can be used in the elements:
     <td class="XML-Token">Cortex-M55</td>
     <td><a href="http://developer.arm.com/ip-products/processors/cortex-m/cortex-m55" target="_blank">Arm Cortex-M55</a> processor based device</td>
   </tr>  
+  <tr>
+    <td class="XML-Token">Star</td>
+    <td><a href="https://www.armchina.com/mountain?infoId=160" target="_blank">ArmChina STAR-MC1</a> processor based device</td>
```
Comment (jkrech) [inline doxygen/src/devices_schema.txt:2375]: I wish we had a url for a web page in english but I could not find one either.

File: doxygen/src/devices_schema.txt [hunk 2]
```diff
@@ -2386,6 +2390,10 @@ These values can be used in the elements:
     <td class="XML-Token">ARMV8MML</td>
     <td>Processor <a href="http://developer.arm.com/products/architecture/m-profile" target="_blank">ArmV8MML</a> compliant with the Armv8-M Mainline Architecture.</td>
   </tr>
+  <tr>
+    <td class="XML-Token">ARMV81MML</td>
```
Comment (jkrech) [inline doxygen/src/devices_schema.txt:2394]: Good catch. This was missing.

File: schema/PACK.xsd [hunk 3]
```diff
@@ -17,13 +17,16 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 
-  $Date:        1. Dec 2020
-  $Revision:    1.7.2
+  $Date:        31. Jan 2022
```
Comment (jkrech) [inline schema/PACK.xsd:20]: We need to update the copyright year 2013-2022

**Review by jkrech — CHANGES_REQUESTED** [review]
Copyright to include 2022.

File: doxygen/src/devices_schema.txt (hunk 1 above)
Comment (JonatanAntoni) [reply doxygen/src/devices_schema.txt:2375]: There seems to be non, unfortunately.

File: schema/PACK.xsd (hunk 3 above)
Comment (JonatanAntoni) [reply schema/PACK.xsd:20]: Good catch. Thanks. Fixed.

**Pull request merged**